src/olympia/activity/models.py

```diff
@@ -312,6 +312,90 @@ def review_log(self):
         )
         return qs
 
+    def create(self, action, *args, **kw):
```

Comment:
The problem I have with this approach is that `<Model>.<manager>.create()` is a standard Django method and we're overriding it with a slightly incompatible method with a different signature (because of the `action` as first arg)

The code does support passing the `action` as an `int` directly though, which _is_ compatible with the standard Django method, so maybe the only thing to do here is to stop making it the first mandatory argument, just having `*args, **kwargs` as the signature, and ensure all callers pass `action` as a keyword argument ?

Reply:
https://github.com/mozilla/addons-server/pull/21484/commits/4d1994af818665eee32102b49e4e51fb3da331e1